creatures.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=AnimeCreatures', 'root', 'password');
+
+function display($db)
+{
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
I would recommend separating the logic for the DB connection into its own separate function. That way, you can reuse it on other pages and do other things besides echoing the collection (as it will only do one thing - create the DB connection).